nixos/modules/services/networking/ssh/sshd.nix

```
@@ -296,6 +296,13 @@ in
         '';
       };
 
+      authorizedKeysInHomedir = mkEnableOption ''
+        the use of the `~/.ssh/authorized_keys` file.
+
+        Otherwise, the only files trusted by default are those in `/etc/ssh/authorized_keys.d`,
+        *i.e.* SSH keys from [](#opt-users.users._name_.openssh.authorizedKeys.keys)
+      '' // { default = lib.versionAtLeast "23.11" config.system.stateVersion; };
```

```suggestion
      '' // { default = lib.versionAtLeast "24.11" config.system.stateVersion; };
```
(this might break setups on 24.05 since people may already use that)